pkg/collect/host_run.go

```diff
@@ -66,14 +119,55 @@ func (c *CollectHostRun) Collect(progressChan chan<- interface{}) (map[string][]
 		return nil, errors.Wrap(err, "failed to marshal run host result")
 	}
 
-	output := NewResult()
 	output.SaveResult(c.BundlePath, resultInfo, bytes.NewBuffer(b))
 	output.SaveResult(c.BundlePath, result, bytes.NewBuffer(stdout.Bytes()))
+	// walkthrough the output directory and save result for each file
+	if runHostCollector.OutputDir != "" {
+		runInfo.OutputDir = runHostCollector.OutputDir
+		bundleOutputRelativePath = filepath.Join(collectorRelativePath, runHostCollector.OutputDir)
+		output.SaveResults(c.BundlePath, bundleOutputRelativePath, cmdOutputTempDir)
+	}
+
+	return output, nil
+}
+
+func (c *CollectHostRun) processEnvVars(cmd *exec.Cmd, runInfo *HostRunInfo) error {
+	guaranteedEnvs := []string{"PATH", "KUBECONFIG"}
+	for _, key := range guaranteedEnvs {
+		guaranteedEnvVal, found := os.LookupEnv(key)
+		if found {
+			cmd.Env = append(cmd.Env,
+				fmt.Sprintf("%s=%s", key, guaranteedEnvVal))
+		}
+	}
+
+	runHostCollector := c.hostCollector
+	if runHostCollector.InheritParentEnvs == true {
```

Comment:
Do you handle `InheritParentEnvs = false`?

Reply:
handled by always setting `cmd.Env`